docs/proposals/unified-backend-architecture/tech-spec.md

```diff
@@ -0,0 +1,239 @@
+# Technical Specification for Unified Backend Architecture
+
+## Author and Document Owner
+
+[Denis Shevchenko](https://iohk.io/team/denis-shevchenko/)
+
+## Status
+
+DRAFT
+
+## User Story
+
+[CHW-35](https://iohk.myjetbrains.com/youtrack/issue/CHW-35)
+
+## Motivation
+
+Currently Daedalus can be treated as a heavyweight client: it works in a pair with Cardano node which stores
+full copy of the blockchain. It leads three problems:
+
+1. Such a heavyweight client must have enough disk space to store growing blockchain.
+2. Cardano node requires quite a lot system resources.
+3. During (first) start heavyweight client must sync (new) blocks and this is long process, even with good
+   internet connection.
+
+These problems make impossible to install such a heavyweight client on a mobile device. The purpose of Unified
+Backend Architecture is to solve these problems.
+
+We propose to create a lightweight Daedalus client. And it is possible to unify the different use cases under a
+single architecture, starting from the crucial observation that both interfacing with an hardware wallet or
```

Comment:
Mention the use cased earlier. Remember that linking to a YT ticket is not visible to 3rd party devs or auditors.